scripts/mods/mod_reforged/dynamic_spawns/unit_blocks/goblin_blocks.nut

```diff
@@ -3,14 +3,12 @@ local unitBlocks = [
 		ID = "UnitBlock.RF.GoblinFrontline",
 		UnitDefs =
 		[
-			{ BaseID = "Unit.RF.GoblinSkirmisherLOW" },
```

Comment:
out of scope for this commit. This, actually impacts the gameplay and is not explained by the commit message